metatron-app/metatron-discovery
# #3765 fix grid alignment error

### Description


**Related Issue** : #3765 


### How Has This Been Tested?

1. Create a grid chart.
2. Check if the sort function works normally.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
